src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLYING_DATA_FIELD = "supply";
+    private static final String BUYING_DATA_FIELD = "buy";
+    private static final int INDEX_OF_DATA_FIELD = 0;
+    private static final int INDEX_OF_AMOUNT_FIELD = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File dataFile = new File(fromFileName);
+        List<String> dataList = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(dataFile))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                dataList.add(line);
+                line = bufferedReader.readLine();
+            }
```

Comment:
let's move reading to a private method as well

this method can return `List<String>`